functions-methods.js

```diff
@@ -33,4 +56,18 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity( email )   {
+
+    const validationEd = email.includes("@");
+    const validationComma = email.includes(",");
+    const validationPoint = email.charAt(email.length -1);
+
+   if( validationEd === true && validationComma !== true && validationPoint !== "." )    {
+       return true;
+   } else return false;
```

Comment:
Je kunt de uitkomst van de expressie meteen teruggeven en tegelijkertijd iets inkorten:
```suggestion
   return validationEd && !validationComma && validationPoint !== ".";
```